composer.json

```diff
@@ -34,55 +34,55 @@
-        "symfony/asset": "5.2.*",
-        "symfony/console": "5.2.*",
-        "symfony/dotenv": "5.2.*",
-        "symfony/expression-language": "5.2.*",
+        "symfony/asset": "5.4.*",
+        "symfony/console": "5.4.*",
+        "symfony/dotenv": "5.4.*",
+        "symfony/expression-language": "5.4.*",
         "symfony/flex": "^1.3.1",
-        "symfony/form": "5.2.*",
-        "symfony/framework-bundle": "5.2.*",
-        "symfony/http-client": "5.2.*",
-        "symfony/intl": "5.2.*",
-        "symfony/lock": "5.2.*",
-        "symfony/mailer": "5.2.*",
-        "symfony/mime": "5.2.*",
+        "symfony/form": "5.4.*",
+        "symfony/framework-bundle": "5.4.*",
+        "symfony/http-client": "5.4.*",
+        "symfony/intl": "5.4.*",
+        "symfony/lock": "5.4.*",
+        "symfony/mailer": "5.4.*",
+        "symfony/mime": "5.4.*",
         "symfony/monolog-bundle": "^3.1",
-        "symfony/notifier": "5.2.*",
-        "symfony/process": "5.2.*",
-        "symfony/property-access": "5.2.*",
-        "symfony/property-info": "5.2.*",
-        "symfony/proxy-manager-bridge": "5.2.*",
-        "symfony/security-bundle": "5.2.*",
-        "symfony/serializer": "5.2.*",
-        "symfony/string": "5.2.*",
-        "symfony/translation": "5.2.*",
-        "symfony/twig-bundle": "^5.2",
-        "symfony/uid": "5.2.*",
-        "symfony/validator": "5.2.*",
-        "symfony/web-link": "5.2.*",
+        "symfony/notifier": "5.4.*",
+        "symfony/process": "5.4.*",
+        "symfony/property-access": "5.4.*",
+        "symfony/property-info": "5.4.*",
+        "symfony/proxy-manager-bridge": "5.4.*",
+        "symfony/security-bundle": "5.4.*",
+        "symfony/serializer": "5.4.*",
+        "symfony/string": "5.4.*",
+        "symfony/translation": "5.4.*",
+        "symfony/twig-bundle": "^5.3",
+        "symfony/uid": "5.4.*",
+        "symfony/validator": "5.4.*",
+        "symfony/web-link": "5.4.*",
         "symfony/webpack-encore-bundle": "^1.11",
-        "symfony/workflow": "5.2.*",
-        "symfony/yaml": "5.2.*",
+        "symfony/workflow": "5.4.*",
+        "symfony/yaml": "5.4.*",
         "twig/extra-bundle": "^2.12|^3.0",
         "twig/twig": "^2.12|^3.0"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^3.1",
         "hautelook/alice-bundle": "^2.8",
-        "symfony/browser-kit": "^5.2",
-        "symfony/css-selector": "^5.2",
-        "symfony/debug-bundle": "^5.2",
+        "symfony/browser-kit": "^5.3",
+        "symfony/css-selector": "^5.3",
+        "symfony/debug-bundle": "^5.3",
         "symfony/maker-bundle": "^1.0",
-        "symfony/phpunit-bridge": "^5.2",
-        "symfony/stopwatch": "^5.2",
-        "symfony/var-dumper": "^5.2",
-        "symfony/web-profiler-bundle": "^5.2"
+        "symfony/phpunit-bridge": "^5.3",
+        "symfony/stopwatch": "^5.3",
+        "symfony/var-dumper": "^5.3",
+        "symfony/web-profiler-bundle": "^5.3"
```

Comment:
I get that they probably will have the right version, but shouldn't we change the constraints to `^5.4`?

Reply:
I've changed to `5.4.*` in https://github.com/itk-dev/naevnssekretariatet/pull/82/commits/886511f8eaf5f0e8e2861ef4d734d44010f4158f.